### PR TITLE
Gdn update semantic search indexes

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR updates the Index Types topic in Semantic Search.

### CODE REVIEW

File: docs/search-views/semantic-search/concepts/index-type.md [hunk 1]
```diff
@@ -5,20 +5,91 @@ title: Index Type
...
+**When to Use:**
+
+- Datasets where there's a need for a trade-off between speed and precision.
+- High-dimensional data spaces.
+- Text-based semantic searches where understanding context and nuances is vital.
+
 ## IVF (Inverted File)
 
-IVF is a traditional indexing method used in information retrieval systems, where each item is associated with a list of documents (or in this case, vectors) where it appears. When used for vector search, IVF is typically combined with some kind of quantization (like product quantization, or PQ) to create a two-level system where the first level groups vectors into coarse clusters, and the second level uses quantization to compress the residual vectors within each cluster. This provides a balance of efficiency and accuracy in the search.
+IVF is a traditional indexing method used in information retrieval systems, where each item is associated with a list of documents (or in this case, vectors) where it appears.
```
Comment (amolth-macrometa) [inline docs/search-views/semantic-search/concepts/index-type.md]: I know this text was already present, but I am not sure it is correct in this context. Where did you get this sentence?

**Review by amolth-macrometa — CHANGES_REQUESTED** [review]

File: docs/search-views/semantic-search/concepts/index-type.md (hunk 1 above)
Comment (amolth-macrometa) [reply docs/search-views/semantic-search/concepts/index-type.md]: This was the only comment I had

Comment (diana-macrometa) [reply docs/search-views/semantic-search/concepts/index-type.md]: Should we change this part?

Comment (amolth-macrometa) [reply docs/search-views/semantic-search/concepts/index-type.md]: yes, just remove that one line

Comment (diana-macrometa) [reply docs/search-views/semantic-search/concepts/index-type.md]: Done. Anything else, or is this ready for approval?

**Review by amolth-macrometa — APPROVED** [review]

**Pull request merged**